### ISSUE TITLE
not able to run

### ISSUE THREAD

**Issue by sychris** [minutelabsio/evolution-simulator#28]
im feeling pretty dumb right now but ive never worked with rust so i installed both rust and wasm open a cmd window and try yarn run. not recognized.  rust web site says to use "cargo run" ok, telling me to specify package or workspace to run and and error a bin target must be set. tried passing it the path, package and workspace not going anywhere. I did get cargo install to do something but no idea what really. could I get some instruction that has a tad more detail than "yarn install" please as not being familure with the software used here that doesn't seam to be quite enough.

**Comment by wellcaffeinated** [comment]
Yeah it's a bit cryptic. Basically the evolution simulator is built on top of Vue with a RUST module run as wasm. You can't run the rust code without the website portion. So `yarn install` (or `npm install`) should install the website related stuff, and then as long as wasm-pack is installed you should be able to do `yarn dev` or `npm run dev` and get a development version of the app served on your local machine. 

let me know if you need more info
